__tests__/apis.test.js

```diff
@@ -4,6 +4,7 @@ const app = require('../app.js')
 const db = require("../db/connection.js");
 const seed = require("../db/seeds/seed.js");
 const data = require("../db/data/test-data");
+require("jest-sorted");
```

Comment:
You shouldn't need to require this in here - you may need to look at the jest sorted docs to amend your package.json to set this up properly